core/src/main/java/com/netease/arctic/catalog/BaseArcticCatalog.java

```diff
@@ -331,6 +329,30 @@ protected TableMetaStore.Builder getMetaStoreBuilder() {
         builder.withBase64KrbAuth(keytab, krb5, principal);
       }
     }
+
+    boolean loadAuthFromAMS = PropertyUtil.propertyAsBoolean(properties,
+        CatalogMetaProperties.LOAD_AUTH_FROM_AMS, CatalogMetaProperties.LOAD_AUTH_FROM_AMS_DEFAULT);
+    String authType = properties.get(CatalogMetaProperties.AUTH_CONFIGS_KEY_TYPE);
+    // if disable ams auth configs and no provide auth configs, use process auth configs
+    if (!loadAuthFromAMS) {
+      if (StringUtils.isEmpty(authType)) {
+        builder.withDisableAuth(true);
+      }
+    }
+
+    // cover auth configs from ams with auth configs in properties
+    if (StringUtils.isNotEmpty(authType)) {
+      if (CatalogMetaProperties.AUTH_CONFIGS_VALUE_TYPE_SIMPLE.equalsIgnoreCase(authType)) {
```

Comment:
UnspportedException should be thrown if an unspported authType set by user.

Reply:
The check has already exist in TableMetastore class